python/sglang/srt/models/deepseek_v2.py

```diff
@@ -1007,6 +1012,17 @@ def forward_absorb_fused_mla_rope(
         return output
 
 
+class _DecoderLayerExecutionMode(Enum):
```

Comment:
Are there better name options here? `MLP_ONE` and `MLP_ALL` are difficult to immediately understand their purposes.

Reply:
renamed